src/PVOutput.Net/PVOutput.Net.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
@CodeCasterNL 
I don't think this file is needed, it looks like it's generated by JetBrains Rider? Could you verify this and delete the file if it isn't essential?

Reply:
It's not needed, but it prevents ReSharper (and possibly Rider as well) from complaining about the interfaces in the Interfaces folder not being in the ...Interfaces namespace, so it prevents a couple of warnings. I can remove it if you want.

Reply:
Ah that's a weird warning to create to be honest. But in case other contributors ever run in to the same issue, it's probably no problem just to leave it there. I work with VS2019 so I can't maintain it myself but that's okay.

Just leave the file there :) If I can find the time tonight, I'll merge the changes in to `develop`.